Templates/MonoGame.Templates.VSExtension/Templates.pkgdef

```diff
@@ -1,2 +1,2 @@
-[$RootKey$\TemplateEngine\Templates\MonoGame.Templates.VSExtension\3.8.1.1]
+[$RootKey$\TemplateEngine\Templates\MonoGame.Templates.VSExtension\3.8.2.1]
```

Comment:
@SimonDarksideJ  ?

Reply:
@CartBlanche any thoughts?

Reply:
In review, yeah this SHOULD not be changed by any other PR than a release, please revert